test/integration/project-management.spec.ts

```diff
@@ -49,19 +49,27 @@ describe('admin.projectManagement', () => {
 
   describe('listAndroidApps()', () => {
     it('successfully lists Android apps', () => {
-      return admin.projectManagement().listAndroidApps().then((results) => {
-        expect(results.length).to.be.at.least(1);
-        expect(results[0].appId).to.equal(androidApp.appId);
-      });
+      return admin.projectManagement().listAndroidApps()
+          .then((apps) => Promise.all(apps.map((app) => app.getMetadata())))
+          .then((metadatas) => {
+            expect(metadatas.length).to.be.at.least(1);
+            const metadataOwnedByTest =
+                metadatas.find((metadata) => isIntegrationTestApp(metadata.packageName));
+            expect(metadataOwnedByTest.appId).to.equal(androidApp.appId);
```

Comment:
Nit: May be check `metadataOwnedByTest` is not `undefined` before comparing `appId`.

Reply:
Done.